Makefile-py

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
can this be in the folder with all the python stuff? or rename it to just `Makefile` and i'll start adding rando scripts here too

Reply:
My thought was that if other Make commands were introduced you could create one Make file that combines them all. It can be useful to separate out the docker specific Make stuff using something like `make -f Makefile-docker` but also combine them with a core Makefile and use `include Makefile-docker`. I thought the separation might help but maybe not. 🤷‍♂️

Reply:
having sub-makefiles and composing them sounds like a cool pattern, in that case could we move the Makefile-py into a python subdirectory? or would that mess with the cwd of the scripts and be confusing? i'm not _opposed_ to having stuff in the top level, i'd just like to put things in subdirectories that can be.

Reply:
yes we could move the py Makefile to a subdirectory.

Reply:
I moved it. This is resolved.